.protolint.yaml

```diff
@@ -0,0 +1,58 @@
+---
+# Lint directives.
+lint:
+  # Linter directories to walk.
+  directories:
+    # The specific directories to exclude.
+    exclude:
```

Comment:
There's no way to define the directories to _include_ instead of the ones to _exclude_ is there? From a quick look I didn't find anything in the example config.

Reply:
I have not found an example for including dirs in the config yet. Protolint required a param passed for what to lint. So, I am passing a dot `protolint lint .`

Reply:
Maybe it's considered the included dirs?

Reply:
If I don't find how to add specifc paths to the config, I will add them as the positional argument.

Reply:
> Maybe it's considered the included dirs?

Could be 🤔

Reply:
Added the `/api` directory as a positional argument to the lint command (tested that it picks up errors in this dir recursively) and removed the directories to exclude from the config.